Tests/kaas/kaas-sonobuoy-tests/README.md

```diff
@@ -34,21 +34,20 @@ For test development with Sonobuoy, [KinD](https://kind.sigs.k8s.io/) is used as
     make dev-setup
     ```
 
-1. Set environment variables
+* (Optional): Set env
 
     ```bash
-    export IMAGE_VERSION_TAG="dev"
-    export K8S_HOST=<kind-cluster-ip>
-    export K8S_PORT=<kind-cluster-port>
+    export TEST_STANDARD=<standard number code>
```

Comment:
I am just curious, what changes when I export  `TEST_STANDARD` env variable?

Reply:
I'm sorry that this is an artifact of the first commit.
The purpose of this variable is to introduce a method to execute tests only for the test functions you are developing.
So far I have developed this part and only recently added a new commit to this merge request regarding its handling.
This variable is now called `TESTFUNCTION_CODE` 
Together with `make test-function` you can now only execute certain test functions.
FYI I also updated the README.md regarding its handling: (see: [README.md#84](https://github.com/SovereignCloudStack/standards/pull/761/files#diff-3087cdc8f309b74c3aeef3698392491f62d4573b22757c7127bd7c7916689f99R84))
 Since this works outside the scope of sonobuoy, it also skips the process of creating the Docker image.

However, the currently implemented setup is based on the creation of k8s clusters by the tessuite whit the usage of kind.
So to speed up the testing process even more, we need to implement a method that allows to use already existing k8s clusters.
The framework should provide methods for this, but I haven't managed to implement it yet. 

I would postpone this part to a subsequent issue and currently use
`Not Yet Implemented` notes as placeholders.
 (here:  [main_test.go#57](https://github.com/SovereignCloudStack/standards/pull/761/files#diff-8cd706634d480289ceb6ac45964e12c5c0801c4f0d6b715a299c631d74e6c69aR57)) 


Example output of using the `make test-function` target:

```
tf@tf-ThinkPad:~/standards/Tests/kaas/kaas-sonobuoy-tests$ export TESTFUNCTION_CODE="0200"
tf@tf-ThinkPad:~/standards/Tests/kaas/kaas-sonobuoy-tests$ time make test-function 
only run tests for: 0200
DEVELOPMENT_MODE=createcluster go test -run=${TESTFUNCTION_CODE}  ./... || true
2024/09/30 12:01:05 Setup test enviornment for: "createcluster"
2024/09/30 12:01:05 Create kind cluster for test
2024/09/30 12:01:05 Creating kind cluster  gotestcluster-02
2024/09/30 12:01:05 launching: kind create cluster --name gotestcluster-02
2024/09/30 12:01:32 kind clusters available:  gotestcluster-02
testcluster
--- FAIL: Test_scs_0200_smoke (0.00s)
    scs_0200_smoke_test.go:14: developmentMode is set to = createcluster; want None
FAIL
2024/09/30 12:01:39 Destroying kind cluster  gotestcluster-02
2024/09/30 12:01:40 Removing kubeconfig file  /tmp/kind-cluser-gotestcluster-02-kubecfg491019238
FAIL	kaas/kaas-sonobuoy-tests/scs_k8s_conformance_tests	34.871s
FAIL

real	0m37,957s
user	0m3,412s
sys	0m1,869s

```

Reply:
thanks for the example.. would be really great, if we could have a little example-walk-through in the README to speed up the familiarisation process

Reply:
I agree that will be helpful. 
So in the past we had decided to put a more detailed step-by-step guide in the Docs repository.
This is also currently being worked on: https://github.com/SovereignCloudStack/docs/pull/221
If you want to see the current version, have a look at the following file:
[conformance-tests-sonobuoy.md](https://github.com/SovereignCloudStack/docs/blob/201-add-guidelines-to-write-good-conformance-tests-for-k8s/contributor-docs/development/tests/conformance-tests-sonobuoy.md)

Reply:
Thank you for the explanation and the given example. I will definitely try it as this seems to be a cool and useful optional variable.